config/webpack.dev.js

```diff
@@ -52,7 +52,14 @@ module.exports = merge(
       },
       watchFiles: {
         paths: ['src/**/*', 'public/**/*']
-      }
+      },
+      proxy: [
+        {
+          context: ['/credentials', '/jobs', '/ping', '/reports', '/sources', '/scans', '/status', '/token', '/users'],
+          target: 'http://0.0.0.0:8000',
```

Comment:
Should this be `127.0.0.1`?

Reply:
So kinda... [these are pointed at what was happening in v2 as part of the container allowance, very similar to the dotenv settings in v2](https://github.com/quipucords/quipucords-ui/blob/v2-alpha/.env.development#L6)... webpack itself needs to be pointed at 127.0.0.1. This one I have to sort-of wait to confirm our Linux user can access it

Reply:
Looks like a confirmation we can skip this update